aip/0217.md

```diff
@@ -12,9 +12,9 @@ redirect_from:
 
 Occasionally, a user may ask for a list of resources, and some set of resources
 in the list are temporarily unavailable. For example, a user may ask to list
-resources across multiple locations, but one of those locations is temporarily
-down. In this situation, it is still desirable to give the user as much as
-possible, while indicating that something is missing.
+resources across multiple parent locations, but one of those locations is
+temporarily unreachable. In this situation, it is still desirable to give the
+user as much information as possible, while indicating that something is missing.
```

Comment:
"As much information as possible" would include "information about what's wrong" - but we specifically exclude that.

How about "all the reachable resources"?

Reply:
That's fair. SGTM.